.pullapprove.yml

```diff
@@ -0,0 +1,83 @@
+version: 3
+
+groups:
+  # Extensions
+  # Handles any changes related to extensions, web store, and chrome apps.
+  # By default Simeon and Andy will be tagged in as reviewers, but anyone
+  # on the dcc-extensions, dcc-content, or dcc-approvers team can approve PRs.
+  content:
+    reviewers:
+      users:
+      - awfuchs
+      - dotproto
+      teams:
+      # Prefix a username or team with ~ to include them in the reviewer pool
+      # but skip them when sending review requests.
+      - ~dcc-extensions
+      - ~dcc-content
+      - ~dcc-eng
+      - ~dcc-approvers
+    reviews:
+      # Assign all possible reviewers
+      request: 99
+    conditions:
+      - >
```

Comment:
It seems like these rules only exist to basically say _NOT_ the old nacl or mobile client docs. Is that right?

Reply:
Essentially. Though when we add devtools, workbox, etc, we'll probably need a similar block for them.

btw here's Angular's pullapprove file. They're like expert mode: https://github.com/angular/angular/blob/master/.pullapprove.yml

Probably some good ideas we can steal from them.